docs/reference/connectors/gcs_writer.md

```diff
@@ -0,0 +1,80 @@
+---
+sidebar_label: gcs (Google Cloud storage)
+sidebar_position: 1
+---
+
+# The Google Cloud Storage Connector
+
+This connector provides the ability to write into Google Cloud Storage.
+
+## Authentication
+This connector will use credentials stored in a JSON file pointed to by the `GOOGLE_APPLICATION_CREDENTIALS` environment variable.
+Alternatively, if tremor is running inside Google Cloud, the authentication will automatically use the credentials of the machine it is running on.
+
+## Writer

```

Comment:
pls document the behaviour of the connector that it will finish an upload if a different `name` - `bucket` pair comes in via an event or tremor stops. So if we have events coming in unordered (e.g. `bucket1 - name1`, `bucket1 - name2`, `bucket1 - name`), object can be overwritten. I do think this is important information. (If we don't have it documented in the s3 connector as well, we should do it quickly).